model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+groups:
+  - id: metric.telemetry.sdk.trace.spans.ended
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.ended
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.spans.sampled
```

Comment:
I'm not really happy with this metric. I'd rather have a `sampled` boolean attribute on the `metric.telemetry.sdk.trace.spans.ended` metric.

However, do we have a way of defining "local" attributes which are not part of the global namespace, such as a simple `sampled` attribute? I couldn't find any example for this. Or do we have to go through the process of defining a globally unique attribute name for this?